app/assets/javascripts/dreamsis.js

```diff
@@ -110,5 +113,8 @@ $( function() {
     );
   })
   
+  // Enable all tablesorter tables
+  // $("table:not(.no-sort)").tablesorter()
```

Comment:
Yes - but need to spend a little more time with it. I was hoping to globally enable it but it majorly messes up some things. Will probably just manually enable on a per-table basis. I was thinking using class name "sortable."

Also need to work on the styling a bit — but it should be a pretty easy integration.